About.html

```diff
@@ -0,0 +1,184 @@
+          <div class="thumbnail border p-5 bg-light">
+            <a href="https://support.microverse.org/en/articles/1908266-what-will-i-learn-at-microverse">  <img src="murple_logo.png" class="img-fluid w-50" alt="Sample1"></a>
+          
+            <div class="caption">
+              <p class="mb-2 text-secondary" style="color: rgb(108, 117, 125); background-color: rgb(248, 249, 250);">2020</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="thumbnail border p-5 bg-light">
+            <a href="https://www.freecodecamp.org/"><img src="./assets/images/Freecode.png" class="img-fluid w-50" alt="Sample1"></a>
+            
+            <div class="caption">
+              <p class="mb-4 text-secondary">2020</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="events" class="py-5 px-3 bg-grey">
+    <div class="text-center">
+      <h2>Bootcamp Events</h2>
+      <hr class="red">
+      <div class="row g-3">
+        <div class="col-md-6">
+          <div class="d-flex flex-column align-content-center">
+            <div class="layer">
+              <h2 class="pt-5">Hacker Rank</h2>
+              <h2>Technical Interviews</h2>
+              <h2 class="pb-5">2023</h2>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="layer">
+            <h2 class="pt-5">Code Wars</h2>
+            <h2>Supports 29 core languages !</h2>
+            <h2 class="pb-5">2023</h2>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="layer">
+            <h2 class="pt-5">Leet Code</h2>
+            <h2>Google, Facebook, Amazon, and Microsoft.</h2>
+            <h2 class="pb-5">2024</h2>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="layer">
+            <h2 class="pt-5">Code Chef</h2>
+            <h2>Typical algorithmic challenges</h2>
+            <h2 class="pb-5">2024</h2>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="sponsor-list" class="bg-dark">
+    <div>
+      <h2 class="text-center">Partners</h2>
+      <hr class="red">
+      <ul class="d-flex gap-5 justify-content-center pt-4">
+          <li>
+            <div class="thumbnail">
+              <img src="murple_logo.png" alt="" class="img-fluid">
+            </div>
+          </li>
+          <li>
+            <div class="thumbnail">
+              <img src="assets/images/Freecode.png" alt="" class="img-fluid">
+            </div>
+          </li>
+          <li>
+            <div class="thumbnail">
+              <img src="assets/images/Google.png" alt="" class="img-fluid">
+            </div>
+          </li>
+          <li>
+            <div class="thumbnail">
+              <img src="assets/images/tesla.png" alt="" class="img-fluid">
+            </div>
+          </li>
+      </ul>
```

Comment:
- [ ] Kindly ensure your background color matches the design as previously requested given below is a comparison


Behance Design          |  Your design     
:-------------------------:|:-------------------------: 
![image](https://user-images.githubusercontent.com/61408860/217659145-5382de79-74fe-4c04-877f-e8011c666bbb.png) |  ![image](https://user-images.githubusercontent.com/61408860/217659254-9620a74b-ba5e-4d59-9c5f-4f45b157ac08.png)